### ISSUE TITLE
Configurable default ingress service http_status:400 

### ISSUE THREAD

**Issue by juliusiqbal** [cloudflare/helm-charts#70]
I was trying to use my own ingress service instead of http_status:400. There is no way to do that with current implementation. Either add default service wtih ingress map or separate variable to change or modify.

https://github.com/cloudflare/helm-charts/blob/b85225eb49e1b0b40328c70d4541a27ec5e2b82e/charts/cloudflare-tunnel/templates/configmap.yaml#L29

Below is what I do now to update value on the fly and after that manually remove pod.
```
kubectl get cm/cloudflare-tunnel -n cloudflare-tunnel -o yaml |
        sed 's/http_status:404/http:\/\/nginx-ingress-controller.nginx-ingress.svc.cluster.local.:80/g' |
        kubectl apply -f -
```

**Comment by lexfrei** [comment]
implemented in #69
currently, you can use it from my repo: https://lexfrei.github.io/charts

**Comment by juliusiqbal** [comment]
Thanks @lexfrei

**Comment by NeuronButter** [comment]
Is there any update on this being merged in? It's been a few months just waiting for one last review...

**Comment by lexfrei** [comment]
@NeuronButter this repo looks abandoned. You can use it from my repo as mentioned above. Here is the argo-cd installation example https://github.com/lexfrei/k8s/blob/4b7c1e11c38eb63bbad1c871dc445fd58c561a7c/argocd/infra/cloudflared.yaml

**Comment by NeuronButter** [comment]
Yeah seems like there are also a few people (myself included) who would be willing to keep it going but we seem to be at the graces of CF employees with free time 😬
